src/SFA.DAS.EmployerIncentives.Api.AcceptanceTests/Pipeline/HealthCheckTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc.Testing;
+using NUnit.Framework;
+
+namespace SFA.DAS.EmployerIncentives.Api.AcceptanceTests.Pipeline
+{
+    [TestFixture]
+    [Parallelizable]
+    public class WhenCheckingPipeLine
+    {
+        [Test]
+        [Ignore("being reworked")]
+        public async Task CallPingEndpoint_ThenShouldReturnOkResponse()
+        {
+            var f = new HealthCheckFixture();
+            var r =  await f.Client.GetAsync("/ping");
+            r.StatusCode.Should().Be(HttpStatusCode.OK);
+        }
+
+        [Test]
+        [Ignore("being reworked")]
```

Comment:
Why is this ignored? The test should have been updated to accommodate the rework

Reply:
as above